### DISCUSSION TITLE
having both public and private IPs for static nodes

### DISCUSSION THREAD

**Discussion by smulikHakipod** [berops/claudie#1353] [General] [answered]
Hey!
I have a on-prem/self-hosted rack with multiple servers, each server have its own 1gbe to WAN with its own IP, but also much faster Lan for internal communication. 

Can I use the Lan for internal communication and the WAN for external one?

For example inter pod commnucation/storage could use the Lan between local nodes, saving bandwidth cost and not overloading the WAN line. Is it possible?

Thanks!

**Comment by bernardhalas** [comment, marked as answer]
Hello @smulikHakipod , currently this mode is not supported.
However, if the public IPs are assigned directly on the WAN interfaces, what you can do is to configure the internal routing tables so that the Public Server1 -> Public Server2 traffic goes via LAN.

e.g.
Server 1: Private IP: 10.0.0.1/24 & Public IP: 5.5.5.1/24
Server 2: Private IP: 10.0.0.2/24 & Public IP: 5.5.5.2/24

Routing table Server 1: Reach 5.5.5.2 via 10.0.0.2
Routing table Server 2: Reach 5.5.5.1 via 10.0.0.1

**Comment by bernardhalas** [comment]
Btw, does your stack support IPv6? We're considering adding the support for IPv6, but to prioritize this properly we'd like to know if our user-base runs stacks that are IPv6 enabled.

**Comment by smulikHakipod** [comment]
Thanks @bernardhalas !
I will try to use to routing trick, thanks for that. 
Regarding IPv6, its good idea. On paper the hardware suppose to support ipv6 (still did not test the ISP side of things), but I never tested it. If can I run something to test it, it would be great. Thanks!

**Reply by bernardhalas** [reply]
Currently, Claudie doesn't support IPv6, but if there's enough interest, we can prioritize it accordingly.
